docs/changelog.yaml-format.md

```diff
@@ -58,7 +58,7 @@ The following is an example of release information for version `1.0.0`:
 ```.yaml
 releases:
   1.0.0:
-    release_date: 2020-04-01
+    release_date: "2020-04-01"
```

Comment:
```suggestion
    release_date: '2020-04-01'
```
Since all other quoted YAML strings in the example use single quotes :)

Reply:
Done.